src/components/About/About.tsx

```diff
@@ -1,128 +1,108 @@
+      thing often, the results may change--hopefully for the better as we refine
+      how the search works. There may be errors. The cross-referencing is
+      automated and based on millions of library records including some that are
+      inaccurate. Part of what we are exploring is how to detect and adjust
+      mistakes, and how to make sure we don’t introduce any new ones.
+    </p>
 
-          <p>
-            It means that this is a work-in-progress. We are constantly working
-            on the interface and trying new ideas to deal with the data. If you
-            visit repeatedly, sometimes it may look different. If you search for
-            the same thing often, the results may change--hopefully for the
-            better as we refine how the search works. There may be errors. The
-            cross-referencing is automated and based on millions of library
-            records including some that are inaccurate. Part of what we are
-            exploring is how to detect and adjust mistakes, and how to make sure
-            we don’t introduce any new ones.
-          </p>
+    <p>
+      It also means that the project may change radically. We may change the
+      URL. We may learn that a different approach is necessary. We may learn
+      that it isn’t useful enough to anyone to continue. If you find books in
+      Digital Research Books Beta that are especially useful to you, you should
+      download a copy so that you have one no matter what becomes of this
+      project.
+    </p>
 
-          <p>
-            It also means that the project may change radically. We may change
-            the URL. We may learn that a different approach is necessary. We may
-            learn that it isn’t useful enough to anyone to continue. If you find
-            books in Digital Research Books Beta that are especially useful to
-            you, you should download a copy so that you have one no matter what
-            becomes of this project.
-          </p>
+    <p>
+      Most of all, it means your feedback is important! Most pages have a
+      feedback button in the bottom right corner. We want to know what you
+      think. If there are things you like or dislike, if there’s a feature
+      missing, if you find an error please tell us in the feedback!
+    </p>
+  </>
+);
 
-          <p>
-            Most of all, it means your feedback is important! Most pages have a
-            feedback button in the bottom right corner. We want to know what you
-            think. If there are things you like or dislike, if there’s a feature
-            missing, if you find an error please tell us in the feedback!
-          </p>
-        </TemplateContentPrimary>
-      </TemplateContent>
-      <TemplateFooter />
-    </Template>
+const About: React.FC = () => {
+  return (
+    <DrbTemplate
+      breadcrumbsDataProps={[{ url: "/about", text: "About" }]}
```

Comment:
let's just call this `breadcrumbsData` if possible. the props is a little redundant i'm realizing (my bad on the naming here).